.github/workflows/ci.yml

```
@@ -96,6 +96,32 @@ jobs:
             packages/babel-standalone/*.js
             !**/node_modules/**
 
+  build-windows:
+    name: Build Babel Artifacts On Windows
+    runs-on: windows-latest
```

```suggestion
    runs-on: windows-latest
    needs: prepare-yarn-cache
```

so that it downloads the cached packages from the previous step.

---

Ehh, it won't restore from that step anyway. The `actions/setup-node` generates different cache id from different os
https://github.com/actions/setup-node/blob/146c4d84a51256d5571b40011b76c0c6f7dfd981/src/cache-restore.ts#L39
even if Yarn 3 does support cache created from different OS.

---

Oh ok